src/MongoMemoryReplSet.ts

```diff
@@ -123,9 +133,10 @@ export default class MongoMemoryReplSet extends events.EventEmitter {
    * Returns a mongodb: URI to connect to a given database.
    */
   async getUri(otherDb?: string | boolean): Promise<string> {
-    if (this._state === 'init') {
+    // TODO : Does this case exists ? this should give an undefined access in this function if we pass nothing
```

Comment:
@nodkz Like described in the TODO comment, does this case even happen ? And if yes it might throw undefined value access error if we pass no parameter to this function.

Reply:
Cannot help with this question, this class was written by @jloveridge
Maybe @jloveridge can help with this code? 

Blame: https://github.com/nodkz/mongodb-memory-server/blame/master/src/MongoMemoryReplSet.js

Reply:
Return this await back.
We should not return `uri` while ReplicaSet is not ready.